dasgo/responses.go

```diff
@@ -40,9 +40,11 @@ type CurrentAuthorizationInformationResponse struct {
 	Application *Application `json:"application"`
 	Scopes      []*int       `json:"scopes"`
 	Expires     *time.Time   `json:"expires"`
-	User        *User        `json:"user"`
+	User        *User        `json:"user,omitempty"`
 }
 
+// TODO: not in documentation not sure if should have omitempty
```

Comment:
The url field:
```json
{
  "url": "wss://gateway.discord.gg/"
}
```
Doesn't really matter cause the client never marshals this. If it's empty during an unmarshal, nothing happens (and there is a problem with Discord). Not valid as a zero value so including omitempty is fine.